src/utils/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { getNetworks } from "./networks";
+import { ethers } from "ethers";
+
+const networks = getNetworks();
+
+export const graphqlEndpoint = "http://localhost:5555/graphql";
+
+export const convertChainIdToNetworkName = (chainId: number) => {
+  return `${networks[chainId].name} (${chainId})`;
+};
+
+export function classNames(...classes: any[]) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export function formatAmount(amount: number) {
+  return ethers.formatEther(amount);
```

Comment:
maybe we just copy the function and define/ So that we don't have to import the entire library ?